packages/core/docs/scripts/index.js

```diff
@@ -0,0 +1,87 @@
+const path = require('path');
```

Comment:
i think those scripts should be outside of "docs" folder. Perhaps in the root/scripts as its something that runs for every package but is not a package by itself.